modules/040-control-plane-manager/images/kube-controller-manager/werf.inc.yaml

```diff
@@ -14,20 +14,6 @@ import:
   add: /src/_output/bin/kube-controller-manager
   to: /usr/bin/kube-controller-manager
   before: setup
-shell:
-  beforeInstall:
-  - apt-get update
-  - apt install ca-certificates apt-transport-https gnupg gnupg2 gnupg1 curl -y
-  - curl -sL 'https://download.ceph.com/keys/release.asc' | apt-key add -
-  - echo deb https://download.ceph.com/debian-pacific/ bionic main | tee /etc/apt/sources.list.d/ceph.list
-  - apt-get update
-  - apt-get install -y ceph-common
-  - touch /etc/ceph/ceph.conf /etc/ceph/ceph.keyring
-  - apt-get purge -y --auto-remove curl apt-transport-https gnupg gnupg2 gnupg1
-  - rm -rf /var/lib/apt/lists/*
```

Comment:
it's safe to remove ceph certs ?

Reply:
it is safe to remove ceph-common from k8s version 1.24 and more

Reply:
Ceph CSI is used now, no need to hack a ceph and rbd binary onto the kube-controller-manager image.

Reply:
we can have installations of k8s 1.23 with old ceph implementation. For k8s 1.23 e should install ceph-common package